src/commands/createNewProject/createNewProject.ts

```diff
@@ -69,6 +69,10 @@ export async function createNewProject(
         await projectCreator.addNonVSCodeFiles();
 
         await initProjectForVSCode(actionContext, functionAppPath, language, runtime, projectCreator);
+        // Functions has a dependency on C# that creates build artifacts: https://github.com/Microsoft/vscode-azurefunctions/issues/658
+        if (language !== ProjectLanguage.CSharp) {
```

Comment:
I thought this was for C#, not for everything but C#...?

Reply:
We want to hide these folders for non-C# projects.  The func cli would create a `bin` and `obj` folder in Node and Python projects which was confusing users at Ignite.